clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/api/v2/client/DockerRegistryClient.groovy

```diff
@@ -92,16 +97,21 @@ class DockerRegistryClient {
       return this
     }
 
+    Builder insecureRegistry(boolean insecureRegistry) {
```

Comment:
I see why the halyard build failed - we need to release this change to clouddriver once it's merged - I'll push it through